common/components/ui/Modal/ModalBody.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import closeIcon from 'assets/images/close.svg';
+
+export default class ModalBody extends React.Component<any, any> {
+  private modal: HTMLElement;
+  private modalContent: HTMLElement;
+  private focusedElementBeforeModal: HTMLElement;
+  private focusableElementsString: string;
+  private focusableElements: HTMLElement[];
+  private firstTabStop: HTMLElement;
+  private lastTabStop: HTMLElement;
+
+  public componentDidMount() {
+    this.focusedElementBeforeModal = document.activeElement as HTMLElement;
+    // Find all focusable children
+    this.focusableElementsString =
+      'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, [tabindex="0"], [contenteditable]';
+    this.focusableElements = Array.prototype.slice.call(
+      this.modal.querySelectorAll(this.focusableElementsString)
+    );
+
+    // Convert NodeList to Array
+    this.firstTabStop = this.focusableElements[0];
+    this.lastTabStop = this.focusableElements[this.focusableElements.length - 1];
+
+    // Focus first child
+    this.firstTabStop.focus();
+
+    this.modal.addEventListener('keydown', this.keyDownListener);
+  }
+
+  public componentWillUnmount() {
+    document.removeEventListener('keydown', this.keyDownListener);
+  }
+
+  public scrollContentToTop = () => {
+    this.modalContent.scrollTop = 0;
+  };
+
+  public render() {
+    const { title, children, modalStyle, hasButtons, handleClose } = this.props;
+    return (
+      <div
+        className="Modal"
+        style={modalStyle}
+        role="dialog"
+        aria-labelledby="Modal-header-title"
```

Comment:
`aria-labelledby` must be an ID, not a class. You may want to generate a unique ID for this in case there are multiple modals.